lib/LiveUpdate/CMakeLists.txt

```diff
@@ -9,25 +9,24 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
 
 if (NOT CMAKE_TESTING_ENABLED)
-  else()
-    if (NOT CMAKE_BUILD_TYPE)
-      set(CMAKE_BUILD_TYPE Release)
-    endif()
-    if (CONAN_PROFILE)
-      set(CONANPROFILE PROFILE ${CONAN_PROFILE})
-    endif()
-    if(NOT EXISTS "${CMAKE_BINARY_DIR}/conan.cmake")
-       message(STATUS "Downloading conan.cmake from https://github.com/conan-io/cmake-conan")
-       file(DOWNLOAD "https://github.com/conan-io/cmake-conan/raw/v0.13/conan.cmake"
-                     "${CMAKE_BINARY_DIR}/conan.cmake")
-    endif()
-    include(${CMAKE_BINARY_DIR}/conan.cmake)
-    conan_cmake_run(
-      CONANFILE conanfile.py
-      BASIC_SETUP
-      ${CONANPROFILE}
-    )
+  if (NOT CMAKE_BUILD_TYPE)
+    set(CMAKE_BUILD_TYPE Release)
   endif()
+  if (CONAN_PROFILE)
+    set(CONANPROFILE PROFILE ${CONAN_PROFILE})
+  endif()
+  if(NOT EXISTS "${CMAKE_BINARY_DIR}/conan.cmake")
+    message(STATUS "Downloading conan.cmake from https://github.com/conan-io/cmake-conan")
+    file(DOWNLOAD "https://github.com/conan-io/cmake-conan/raw/v0.13/conan.cmake"
+      "${CMAKE_BINARY_DIR}/conan.cmake")
+  endif()
+  include(${CMAKE_BINARY_DIR}/conan.cmake)
+  conan_cmake_run(
+    CONANFILE conanfile.py
+    BASIC_SETUP
+    ${CONANPROFILE}
+  )
```

Comment:
Do we really need this?

Reply:
We sure don't :-) But what happened here is that I bulk removed the check for conanbuildinfo.cmake from everywhere to get the integration tests to build, and was focusing only on that part. Removed the remainder now.